internal/core/bootstrap.go

```diff
@@ -70,6 +71,12 @@ func Bootstrap(config *BootstrapConfig) (exitCode int) {
 
 	rootCmd.SetArgs(config.Args[1:])
 	err = rootCmd.Execute()
+
+	switch err.(type) {
```

Comment:
we should add support for an ExitCode field inside CLI error. and return 1 by default

Reply:
where should we transform `interactive.InterruptError` into a `CliError` ?